common/persistence/persistenceInterface.go

```diff
@@ -887,6 +887,17 @@ func FromDataBlob(blob *DataBlob) ([]byte, string) {
 	return blob.Data, string(blob.Encoding)
 }
 
+// Convert a *Datablob to safe that calling its method won't run into NPE
+func (d *DataBlob) ToNilSafeDataBlob() *DataBlob {
+	if d != nil {
+		return d
+	}
+	return &DataBlob{}
+}
+func (d *DataBlob) GetEncodingString() string {
+	return string(d.Encoding)
+}
```

Comment:
yeah we could define one there. 
But for this PR, having this one is slightly better:
The code is 
```
event.GetEncodingString()
```
Instead of 
```
event.Encoding.String()
```
Unless you think `event.Encoding` also works. I feel like gocql library will not be able to get the value right, and don't want to risk/experiment it here.